t/11_hacluster.t

```diff
@@ -76,6 +76,7 @@ subtest '[script_output_retry_check] Diskless SBD scenario' => sub {
     $hacluster->redefine(record_info => sub { note(join(' ', 'RECORD_INFO -->', @_)); });
     # Just returns whatever you put as command
     $hacluster->redefine(script_output => sub { return $_[0]; });
+    $hacluster->redefine(script_retry => sub { note(join(' ', 'SCRIPT_RETRY -->', @_)) });
```

Comment:
Your change in the function under test is a behavioral change: you like the code to do something new. You can eventually expand the test to guarantee that this new behavior is present.